### PR TITLE
Checking Branch

### PR DESCRIPTION
**Pull request by harshita02**
My first Html and CSS project.

### CODE REVIEW

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUEST CHANGES :recycle: 
## Hi @harshita02   :hand: :hand: 
I am @tuyishimireeric your reviewer for this PR :man_beard:

Great work so far! 👏
Thank you for implementing most of the project requirements 👏  however, there are some issues that you still need to work on to go to the next project, but you are almost there!
you-are-near


## To highlight 🎯

:heavy_check_mark:  Gitflow used. :+1: 
:heavy_check_mark: No Linter errors :+1: 
:heavy_check_mark: Descriptive commit messages :+1: 

## Required Changes ♻️

- [x] As suggested in the project requirements, I think it would be better to add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
image
_The README file to a repository communicates important information about your project. It is always a good practice to add it on each created Repository 👍 👍_

## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test .md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (pinkmoon25) [inline test .md]: Please make sure this file is not commited, anything inside .gitignore is ignored by git when pushing to remote, more details :point_right:  [.gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore)

according to project requirements _**"Make sure that the last file is actually ignored by git and is not present in your GitHub repository."**_

as i can see you have two test.md files and this one has space between `test .md` , try making it `test.md` and see if it works, i would suggest use only single test.md file

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @harshita02 , 

Great work on making the changes requested by a previous reviewer 👏 
I see you have the readme template on your main branch
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
linters are passing :heavy_check_mark: 
Added html and css file :heavy_check_mark: 
Added .gitignore

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test .md (hunk 1 above)
Comment (harshita02) [reply test .md]: Hi, could you please check the code again?

**Review by aimalamiri — APPROVED** [review]
Hi @harshita02, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**